src/app.ts

```diff
@@ -63,6 +65,30 @@ const authenticatedRouter = new Router()
         ctx.state.toMember = midQuery.docs[0];
         return await next();
     })
+    .post('/api/members/:mid/request_invite', async ctx => {
+        const authUid = ctx.state.user.uid;
+        const authMember = await members.doc(authUid).get();
+        try {
+            const newOperation = {
+                creator_mid: authMember.get('mid'),
```

Comment:
If we no longer do creator_mid as a standard part of an operation, then for the special REQUEST_INVITE operation let's move it to `data`

Reply:
also agreed, if this is the mid they're assigning at the start then it's still essential info

Reply:
though any changes to operation structure would probably require migration of existing ones or code that can handle multiple versions, which we don't have nor even want. so, let's table this for a separate PR